docs/src/app_dev/app_val_tokenomics/index.rst

```diff
@@ -0,0 +1,121 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _app_val_tokenomics:
+
+Tokenomics Overview
+===================
+
+Following is an overview of the application and validator tokenomics for
+application developers or validator operators. It builds on the "`Canton
+Coin: A Canton-Network-native payment
+application <https://www.digitalasset.com/hubfs/Canton%20Network%20Files/Documents%20(whitepapers%2c%20etc...)/Canton%20Coin_%20A%20Canton-Network-native%20payment%20application.pdf?__hstc=169870847.16854726061d8b28be85af48c17588c4.1750870506327.1750870506327.1750870506327.1&__hssc=169870847.1.1750870506328&__hsfp=1243925796&_gl=1*1fkzmuj*_gcl_au*MTkyOTE1NjAyNC4xNzUwODcwNTA2*_ga*NDU1NzM2NzgyLjE3NTA4NzA1MDY.*_ga_GVK9ZHZSMR*czE3NTA4NzA1MDUkbzEkZzAkdDE3NTA4NzA1MDUkajYwJGwwJGgw>`__"
+whitepaper and assumes you have read the whitepaper.
+
+Canton Network tokenomics is based on an *Activity Record* which
+identifies a party that performed an action which provides value to the
+network. Creating an activity record and minting the associated Canton
+Coin (CC) are two distinct steps. The creation and minting steps are
+performed in a cycle that is called a *round* which has five phases. In
+the first phase, any fee values for that round are written to the ledger
+(the fees can be obtained from the `Scan State
+API <https://docs.sync.global/app_dev/scan_api/scan_current_state_api.html>`__).
+The second phase is called the *activity summary* and it is when
+activity records are created. The next phase calculates a *minting
+weight* for each kind of activity record which is the share of total CC
```

Comment:
Minting weight would be a way to describe it, but can be confusing with the fact that the recording already happens weighted. I'd suggest to use the terminology that "isssuence-per-activity-weight" is computed.

Here's the complete list of parameters computed for issuance: https://github.com/hyperledger-labs/splice/blob/332e06a7ae9e13fde5bba0bf7dcb059aa36f979e/daml/splice-amulet/daml/Splice/Issuance.daml#L66-L75

Reply:
Mentally add "Weight" at the end of the "PerXyzCoupon" parameters and they start to make sense ;-)

Reply:
For reference see the usage of "weight" in the whitepaper:
<img width="952" height="445" alt="image" src="https://github.com/user-attachments/assets/0b0ecb48-1eb3-44b6-ae62-8d953b184f38" />

Reply:
and the example about minting per weight:
<img width="926" height="151" alt="image" src="https://github.com/user-attachments/assets/0919a08c-0fac-4c8d-85b7-e12ba67b2283" />

Reply:
Done.